docs/docs/recipes/configure-storage-providers/README.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
will we leave this open until it's complete?

Reply:
It's ready now